src/cc/export/helpers.h

```diff
@@ -222,7 +222,15 @@ struct _name##_table_t _name = { .max_entries = (_max_entries) }
 #define cursor_advance(_cursor, _len) \
   ({ void *_tmp = _cursor; _cursor += _len; _tmp; })
 
-char _license[4] SEC("license") = "GPL";
+#ifndef EBPF_LICENSE
```

Comment:
could you drop letter E, please ?
And please add a testcase.

Reply:
Ok.

Reply:
Should the compiler warn the user about adding GPL with a #warning? On one hand, it is not nice that the compiler relicenses the code silently. On the other hand, without the warning the behavior is backwards compatible (won't break -Werror=all, for instance). In this case, I think there should be a big notice in the documentation.

Reply:
yes, you can put something in documentation about the new mechanism to define a different bpf program license.